7.1.0.M3 Release is not published

So, all builds with Grails 4.1.0.M3 will fail with
```
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
     Required by:
         project :
      > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
```

Can confirm.

Downloaded grails-4.1.0.M3.zip, then:

```
PS C:\Users\Bob\Desktop\grails-4.1.0.M3\bin> & .\grails.bat create-app fred
| Application created at C:\Users\Bob\Desktop\grails-4.1.0.M3\bin\fred
PS C:\Users\Bob\Desktop\grails-4.1.0.M3\bin> cd fred
PS C:\Users\Bob\Desktop\grails-4.1.0.M3\bin\fred> & .\grailsw.bat run-app
| Resolving Dependencies. Please wait...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'fred'.
> Could not resolve all files for configuration ':runtimeClasspath'.
   > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
     Required by:
         project :
      > Could not resolve org.grails.plugins:hibernate5:7.1.0.M3.
         > Could not get resource 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'.
            > Could not GET 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'. Received status code 409 from server:

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 16s
| Error Error initializing classpath: Could not GET 'https://oss.jfrog.org/oss-snapshot-local/org/grails/plugins/hibernate5/7.1.0.M3/hibernate5-7.1.0.M3.pom'. Received status code 409 from server:  (Use --stacktrace to see the full trace)
PS C:\Users\Bob\Desktop\grails-4.1.0.M3\bin\fred>
```

Looked around jfrog, didn't find the requisite files.

--- a/grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy
+++ b/grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy
@@ -182,6 +182,10 @@ class GrailsGradlePlugin extends GroovyPlugin {
         'web'
     }
 
+    protected String getDefaultMicronautVersion() {
+        '2.2.1'
+    }
+
     void addDefaultProfile(Project project, Configuration profileConfig) {
         project.dependencies.add('profile', "org.grails.profiles:${System.getProperty("grails.profile") ?: defaultProfile}:")
     }
@@ -470,7 +474,8 @@ class GrailsGradlePlugin extends GroovyPlugin {
             project.configurations {
                 agent
             }
-            project.dependencies.add("runtimeOnly", "io.micronaut:micronaut-inject-groovy")
+            final String micronautVersion = project.properties['micronautVersion']
+            project.dependencies.add("runtimeOnly", "io.micronaut:micronaut-inject-groovy:${micronautVersion?:defaultMicronautVersion}")
             project.afterEvaluate(new AgentTasksEnhancer())
         }
     }

